### DISCUSSION TITLE
Use the ENTER key to confirm name change instead of clicking

### DISCUSSION THREAD

**Discussion by johardi** [redivis/meta#169] [Feature requests]
**What is the problem you'd like us to address**
It's too slow when confirming a new name for a node in a project by doing a click instead of using the ENTER key in the keyboard.

**Describe a solution (if you have one in mind)**
Allowing the ENTER key to make the confirmation will make the renaming process much faster.

**Describe other related products we could check out (if applicable)**
N/A

**Any additional context**
N/A

**Comment by imathews** [comment]
Agreed. We are generally working hard to improve keyboard-based navigation across Redivis

**Comment by sfmcintyre** [comment]
Implemented in our [latest release](https://redivis.com/updates#2023-06-22)
